controllers/auth.controller.js

```diff
@@ -0,0 +1,24 @@
+const User = require("../schema/user.schema");
```

Comment:
Im able to authenticate the user in Aldaine branch but not able to authenticate user here.
revise

Reply:
don't understand the issue
what is here?

Reply:
it could be due to the fact that middlewares was made from the aldaine branch before all changes were made. so if its a case where u testing the branch without pulling changes from the aldaine branch then that may be the reason you seeing that

its working on my end